README.md

```diff
@@ -1,29 +1,54 @@
+[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation software.
+OpenFisca allows users to :
+- Calculate many variables of the tax and benefit system of a country (social benefits and taxes paid by households) given input variables.
+- Simulate the budgetary consequences of a reform and its distributional impact when plugged on a survey.
+Its engine is independent of the country, it is therefore possible to simulate any country.
+For now the main supported country is France.
 
-[![OpenFisca logo](https://www.openfisca.fr/hotlinks/logo-openfisca.svg)](https://www.openfisca.fr/)
+[OpenFisca](https://www.openfisca.fr/) is free and is published under the [GNU Affero General Public Licence](https://www.gnu.org/licenses/agpl.html) version 3 or later.
 
-[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation software.
+## Project Components
 
-It is a free software published under the [GNU Affero General Public Licence](https://www.gnu.org/licenses/agpl.html) version 3 or later.
+OpenFisca is a modular project. Depending on your goals, you will install and interact with one or several of the OpenFisca Components.
 
-## What's the purpose?
+[![OpenFisca schema](https://cdn.rawgit.com/openfisca/openfisca-doc/eb7b9ce6f41d7ab2cb7bac08738cfb67d58c0c36/architecture.svg)](https://github.com/openfisca/openfisca-doc/blob/bird_eye_view/architecture.svg)
 
-OpenFisca allows users to calculate many variables of the tax and benefit legislation of a country (social benefits and taxes paid by households) given input variables.
-The impact of reform projects can also be simulated.
+### Web API
+The Web API lets you access the legislation Parameters and Variables.
 
-When plugged on a survey, OpenFisca can also calculate the budgetary consequences of a reform
-and its distributional impact.
+>**Example**: [Mes Aides](https://mes-aides.gouv.fr) uses the OpenFisca Web API to calculate OpenFisca-France benefits.
 
-Its engine is independent of the country, it is therefore possible to simulate any country.
-For now the main supported country is France.
+- To explore the OpenFisca-France Web API services, use the [French Legistation Explorer](https://legislation.openfisca.fr/)
+- To query the Openfisca Web API in your app, see our [Web API endpoints description](./openfisca-web-api/endpoints.md)
+- To host your own instance of the Openfisca API, go to our [installation documentation](./openfisca-web-api/README.md)
+
+### Extensions Packages
+Extensions add on the capacities of a country-package.
+
+>**Example**: See [Paris extension](https://github.com/sgmap/openfisca-paris) and [Rennes extension](https://github.com/sgmap/openfisca-rennesmetropole) on top of OpenFisca France.
+
+- To install an Extension, head to the [Extensions documentation](./contribute/extensions.md)
+- To create a new Extension, head to the [Extension Template on GitHub](https://github.com/openfisca/openfisca-extension-template)
```

Comment:
The [original comment](https://github.com/openfisca/openfisca-doc/pull/74#discussion_r119320620) is outdated, so I can't reply to it directly and will copy here the open question:

> The "Install/Create new" contruct was meant to parralel the Country Package description.
> Should we :
>
> 1. Have "Install/Create new" lines in both Country and Extension Package descriptions ;
> 2. Only have an "Install" line for both Country and Extension Package descriptions ;
> 3. Have "Install/Create new" lines for the Country Package description and only have a "Install" line for the Extension Package descriptions.
> 
> I like option 1 better. What do you think ?

I don't think that the first page can meaningfully direct the user straight to a template with no prior documentation. I'm thus in favour of option 2 🙂